bindings/python/unicorn/unicorn.py

```diff
@@ -17,12 +17,12 @@
 if _python2:
     range = xrange
 
-_lib_path = os.path.split(__file__)[0]
-_all_libs = (
-    "unicorn.dll",
-    "libunicorn.so",
-    "libunicorn.dylib",
-)
+if sys.platform == 'darwin':
+    _lib = "libunicorn.1.dylib"
```

Comment:
I switched all references to the library name to the versioned name, since that's what you did with capstone. I can switch them all back.